src/DripsHub.sol

```diff
@@ -204,11 +204,17 @@ abstract contract DripsHub {
     }
 
     struct SplitsState {
-        /// @notice Users' splits configuration hashes, see `hashSplits`.
+        /// @notice The user's splits configuration hash, see `hashSplits`.
         bytes32 splitsHash;
-        /// @notice The amount collectable independently from cycles
-        /// The key is the asset ID.
-        mapping(uint256 => uint128) collectables;
+        /// @notice The user's splits balance. The key is the asset ID.
+        mapping(uint256 => SplitsBalance) balances;
+    }
+
+    struct SplitsBalance {
+        /// @notice The not yet split balance, must be split before collecting by the user.
+        uint128 unsplit;
+        /// @notice The already split balance, ready to be collected by the user.
+        uint128 split;
```

Comment:
So, basically the `split` is kind of an `accountBalance`.

Are you envisioning the `receiveDrips` to be callable  as standalone?
If thats the case, we would have to increase this balance here in the splitStruct.

I have the feeling the variable should live somewhere else, however for gas reasons it makes sense to put it next to `unsplit`.

Maybe the `SplitBalance` struct is more like an `AccountBalance` with a `balance` and `unsplit`